ph-child.php

```diff
@@ -357,7 +357,7 @@ public function whitelist_option( $options ) {
 		public function create_menu() {
 			add_options_page(
 				__( 'Feedback Connection', 'ph-child' ),
-				__( 'Feedback', 'ph-child' ),
+				__( 'Project Huddle - Client', 'ph-child' ),
```

Comment:
ProjectHuddle is a single word.
And just name it as "ProjectHuddle"

Reply:
done